lib/main_micro_alp.pm

```diff
@@ -51,7 +51,6 @@ sub load_boot_from_disk_tests {
         loadtest 'installation/bootloader_start';
     } elsif (is_vmware()) {
         loadtest 'installation/bootloader_svirt';
-        loadtest 'installation/bootloader_uefi';
```

Comment:
I do not think this is a good idea. It revealed a bug that gfx payload is mangling unpredictably the screen resolution. Also, this is the only place, where we can change kernel parameters at the boot time.

Reply:
for 1) ok. For 2) are we changing those parameters in any job?

Reply:
2) Not in a stable job, but for debug I use it quite often.

Reply:
Ok, anyway, I think 1) is valid enough... Santi, let's try to see if removing some needles help... I'm on it